### DISCUSSION TITLE
Squash commits at pull requests

### DISCUSSION THREAD

**Discussion by pavloshushkov** [mindsdb/mindsdb#7643] [General]
Hello everybody,

Looking at https://github.com/mindsdb/mindsdb/branches/all?page=1
It's a lot of useless branches.
It will be cool to enable squash commits in prs https://docs.github.com/en/repositories/configuring-branches-and-merges-in-your-repository/configuring-pull-request-merges/configuring-commit-squashing-for-pull-requests

**Comment by ZoranPandovski** [comment]
Thanks, we will clean those. We don't want to squash commits to keep the context and avoid issues when reverting. Each PR reviewer should be responsible for deleting the branch after merging
